Functions.Templates/Templates/EventGridTrigger-CSharp/metadata.json

```diff
@@ -6,7 +6,7 @@
     "category": [
         "$temp_category_experimental"
     ],
-    "categoryStyle": "eventHub",
+    "categoryStyle": "eventGrid",

```

Comment:
Eventhub and eventgrid have same icons now?

Reply:
Yeah I will updating such that they each have their own

Reply:
Yes, updating so they will be different